feat: add card component

irfan-nawaz opened the pull request:
added cards and did minor changes in spacing and positioning
live pre-view link: [https://1lgvq.csb.app/](url)

Commit:
feat: add card component

added cards and fixed minor spacing

Commit:
fix: badge position

mohammed-sas reviewed (changes requested):
all the card vairants have box shadow , prd requirement is that only one variant should have box shadow , mention name of the variants for each card. the dismiss buttons should be on the top right corner

irfan-nawaz commented:
> all the card vairants have box shadow , prd requirement is that only one variant should have box shadow , mention name of the variants for each card. the dismiss buttons should be on the top right corner

hey, Mohammed, I have read the requirements on LMS It is written, cards i.e multiple cards can have a box-shadow, and yes I have tried the cards without keeping and box-shadow but I still prefer that all the cards should have box-shadow, and also the same with dismiss icon I have tried different positions and I intended to keep them there only, as there no specific position mentioned for them on LMS and it is free to the creator,
thanks for your valuable suggestions. I kindly request you to approve if the above ones are only the cases.
thank you

kart-c commented on Components/Card/card.css:
```diff
@@ -0,0 +1,202 @@
+
+.content-row {
+    display: flex;
+    flex-direction: row;
```
Here you can remove `flex-direction: row` as its the default value

irfan-nawaz replied:
👍🏼

kart-c commented on Components/Card/card.css:
```diff
@@ -0,0 +1,202 @@
+
+#colors {
+    display: flex;
+    flex-direction: row;
```
I would suggest the same here

irfan-nawaz replied:
👍🏼

kart-c commented on Components/Card/card.css:
```diff
@@ -0,0 +1,202 @@
+#colors {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```
This is also a default value. Attaching link for reference 

https://css-tricks.com/almanac/properties/j/justify-content/#:~:text=flex%2Dstart%20(default)%3A%20items%20are%20packed%20toward%20the%20start%20line

irfan-nawaz replied:
👍🏼

kart-c commented on index.html:
```diff
@@ -107,6 +108,182 @@
+            <p>Unisex</p>
+            <p>available in colors</p>
+            <div id="colors">
+                <div id="blue"></div>
+                <div id="green"></div>
+                <div id="red"></div>
+            </div>
+            <h5 class="product-price">
+                ₹ 8,995
+            </h5>
+        </div>
+        <div class="icons">
+            <button class="btn_otl_default">Add To Cart</button>
+            <i class="far fa-heart"></i>
+            <i class="fas fa-share-alt"></i>
+            <i class="fas fa-ellipsis-v"></i>
+        </div>
+    </div>
+
+    <div class="container">
+        <figure class="content badge">
+            <img src="https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=880&q=80"
+                alt="Image of bubbles through a microscope." />
+            <figcaption class="title">
+                <h2>Nike Air Max</h2>
+                <p>Soft and Smooth.</p>
+            </figcaption>
+            <i class="far fa-heart"></i>
+            <h4>NEW</h4>
+        </figure>
+        <div class="about">
+            <h5 class="product-description">Sustainable Materials</h5>
+            <p></p>
+            <p>Unisex</p>
+            <p>available in colors</p>
+            <div id="colors">
+                <div id="blue"></div>
+                <div id="green"></div>
+                <div id="red"></div>
```
`id` attribute is supposed to be unique. If you want to use it multiple times you should use `class`

irfan-nawaz replied:
hey Kartikeya thanks for your reviews will make the changes right away 🚀

Commit:
fix: made suggested changes

remove default properties and changed attribute from id to class for common styled elements
